src/containers/PageContainer/elements/Header/elements/InfoButton/elements/InfoPanel/_info-panel.scss

```diff
@@ -0,0 +1,97 @@
+.info-panel {
+	&__container {
+		position: absolute;
```

Comment:
pixels and rem

Reply:
I think we need to use only one of them pixel or rem. I always used rem because it should be responsive in all of screens.